src/compiler/resolutionCache.ts

```diff
@@ -831,6 +861,11 @@ namespace ts {
             filesWithInvalidatedNonRelativeUnresolvedImports = filesMap;
         }
 
+        function invalidateTypingsFiles() {
+            hasChangedAutomaticTypeDirectiveNames = true;

```

Comment:
This is a bit of an abuse of this variable—the _names_ didn’t actually change; the resolution results did. The distinction isn’t important for consumers at the moment, so it’s a shame that the name doesn’t quite fit. I can add a different variable and corresponding method on `CompilerHost` unless there’s a better option.